### ISSUE TITLE
consider browserless download of publishSettings

### ISSUE THREAD

**Issue by tjanczuk** [tjanczuk/git-azure#18]

**Comment by woloski** [comment]
This would be great. Download of the publishSettings is such a confusing process, especially if you have multiple subsriptions and more confusing even if you reached the 10 certs quota. But I'm not sure this is doable since there is no API to do that. You would have to login with windows Live. Are you thinking of using a headless browser and ask for the LiveID user and password?

**Comment by tjanczuk** [comment]
Yes, mimicking the browser and specifying LiveID creds on the command line is what I thought to look into.
